data/news/platform/ocaml-5.0.md

```diff
@@ -0,0 +1,603 @@
+---
+title: Release of OCaml 5.0.0
+description: Release of OCaml 5.0.0
+date: "2022-12-16"
+tags: [ocaml, platform]
+---
+
+We have the pleasure of celebrating the birthday of Arthur C. Clarke by
+announcing the release of OCaml version 5.0.0.
+
+The highlight of this new major version of OCaml is the long-awaited runtime support
+for shared memory parallelism and effect handlers.
+
+This multicore support is the culmination of more than 8 years of effort,
+and required a full rewrite of the OCaml runtime environment.
+Consequently, OCaml 5.0.0 is expected to be a more experimental version of
+OCaml than the usual OCaml releases.
+
+For instance, in this release the native compiler only supports the x86-64 and arm64 architectures. In terms of operating systems, Linux, BSDs, MacOS and mingw64 on Windows
+are supported.
+
+We are planning to restore the support for most previously supported architectures and operating systems, and address remaining known issues over the course of the next year.
+
+Don't let those words of caution hinder your experimentation with OCaml 5:
+OCaml 5 as a language is fully compatible with OCaml 4 down to the performance
+characteristics of your programs. In other words, any code that works with OCaml 4 should work the same with OCaml 5. 
+
+The currently known exceptions to this rule are:
+- the removal of many long-deprecated functions and modules
+- changes to the internal runtime API
+- the performance of ephemerons is currently (and temporarily) strongly degraded.
+
+Please report any unexpected behaviours on the [OCaml issue tracker]
+(https://github.com/ocaml/ocaml/issues).
```

Comment:
"and post any questions or comments you might have on our [discussion forums](https://discuss.ocaml.org)"